pages/Login/index.tsx

```diff
@@ -22,106 +25,138 @@ const Login: NextPage = () => {
   } = useForm<ILogin>();
   const router = useRouter();
   const { data, mutate, ...dataQuery } = useLogin();
+  const [isError, setError] = useState(false);
+  const [value, setValue] = useState(0);
 
   const onSubmit: SubmitHandler<ILogin> = (dataForm) => {
     const success = () => {
       router.push(Routes.Home);
     };
-    mutate(dataForm, { onSuccess: success });
+    const error = () => {
+      setError(true);
+    };
+    mutate(dataForm, { onSuccess: success, onError: error });
   };
 
+  useEffect(() => {
```

Comment:
if we take this approach, we will have to copy this code every time we want to show the loader, right?